tests/dist/src/test/java/org/hawkular/apm/tests/dist/TraceServiceITest.java

```diff
@@ -341,10 +341,10 @@ public void testStoreAndQueryEndTimeExclude() {
 
         // Query stored trace
         Criteria criteria = new Criteria();
-        criteria.setEndTime(1500);
+        criteria.setEndTime(2);
```

Comment:
Time units on criteria were not changed. Either I change time units on trace objects or on here. 

-> I have changed time values on trace objects.